src/Core/Ast/Parser/NikicPhpParser/NikicPhpParser.php

```diff
@@ -35,12 +35,14 @@ class NikicPhpParser implements ParserInterface
 
     /**
      * @param ReferenceExtractorInterface[] $extractors
+     * @param array<mixed> $config
```

Comment:
The shape of the array is known here, no? If should, it should be properly typed.

Reply:
done